README.md

```diff
@@ -22,6 +22,7 @@ open an Issue or Pull Request for that.
 - Install RePoE
   * Clone RePoE and go into its folder
   * install: `pip install -e .`  
+  * install pre-commit: `pre-commit install`
```

Comment:
This command did not work for me. Seems like `pip install pre-commit` is what you're supposed to use for installation.

Reply:
Ah yeah sorry, you're supposed to do both.
Wasn't sure if adding `pre-commit` to the requirements would make it auto install upon pip installing repoe

Reply:
Ah, that should happen when pip installing repoe. Since repoe was already installed for me, I only tried `pre-commit install`, which obviously failed.